lib/ast.js

```diff
@@ -629,9 +629,14 @@ var AST_Switch = DEFNODE("Switch", "expression", {
     }
 }, AST_Block);
 
-var AST_SwitchBranch = DEFNODE("SwitchBranch", null, {
+var AST_SwitchBranch = DEFNODE("SwitchBranch", "body", {
     $documentation: "Base class for `switch` branches",
-}, AST_Block);
+    _walk: function(visitor) {
```

Comment:
I'm not in favor of these changes to the definition of AST_SwitchBranch. I'd recommend to revert lib/ast.js and use the patch outlined in https://github.com/mishoo/UglifyJS2/pull/1334#issuecomment-254912896